libs/api/Verifier.cpp

```diff
@@ -212,7 +237,7 @@ _get_map_names(
         _parse_btf_map_info_and_populate_cache(reader, map_names);
     }
 
-    if (get_map_descriptor_size() != map_names.size()) {
+    if (get_map_descriptor_size() < map_names.size()) {
```

Comment:
Can it be `>`?  If so, how can you reliably know that a map name was not found for some maps?